Day 7/hangman.py

```diff
@@ -0,0 +1,48 @@
+import random
+import hangman_words
+from hangman_art import stages, logo
+
+word_list = hangman_words.word_list
+lives = 6
+
+print(logo)
+
+chosen_word = random.choice(word_list)
+world_length = len(chosen_word)
+
+# create switch
+end_of_game = False 
+
+# create blanks 
+display = []
+for letter in range(world_length):
```

Comment:
letter ?? 
world_length include int data type as I suppose from name. Try print(letter) and check is this variable has a really letter or something else.